test/libdnf5/base/test_transaction.cpp

```diff
@@ -45,7 +45,7 @@ void BaseTransactionTest::test_check_gpg_signatures_no_gpgcheck() {
 void BaseTransactionTest::test_check_gpg_signatures_fail() {
     add_repo_repomd("repomd-repo1");
 
-    base.get_config().get_gpgcheck_option().set(true);
+    base.get_config().get_pkg_gpgcheck_option().set(true);
```

Comment:
Do we have at least some simple test checking that both options behave in the same way? Either a CI or a unit test ...